lib/commands/alert.js

```diff
@@ -19,6 +21,24 @@ commands.getAlertText = async function () {
   }
 };
 
+helpers.nativeSetAlertText = async function (text) {
+  try {
+    // first make sure that there is an alert
+    await this.findNativeElementOrElements('class name', 'XCUIElementTypeAlert', false);
```

Comment:
This verification is not needed now, since the class chain query below won't find anything if there is no parent alert

Reply:
It is needed by the specification, since we need to throw a no-alert error if there is no alert.